geometry_msgs/PoseArray eye_data_list
geometry_msgs/PoseArray hand_data_list 
---
geometry_msgs/Quaternion[] rt_list